datumaro/components/dataset.py

```diff
@@ -853,6 +865,19 @@ def get(self, id: str, subset: Optional[str] = None) -> Optional[DatasetItem]:
     def get_annotated_size(self):
         return self._data.get_annotated_size()
 
+    def get_subset_info(self):
+        return "".join(f"\t{subset_name}: # of items={len(self.get_subset(subset_name))}, "
```

Comment:
`\t` seems to go upper level because we can use this function as a standalone.

Reply:
I set `separator='\t'` to concatenate them. If only remove `\t` in here, the tab does not apply to the next line.